packages/router/src/util.ts

```diff
@@ -445,6 +443,14 @@ interface AnayzeRoutesOptions {
 
 type WhileLoadingPage = () => ReactElement | null
 
+// Not using AvailableRoutes because the type is generated in the user's project
+// We can't index it correctly in the framework
```

Comment:
```suggestion
// Not using AvailableRoutes. Because the type is generated in the user's
// project we can't index it correctly in the framework
```

Reply:
I don't think the grammar on the suggestion is correct ;) Leaving as is.